### PR TITLE
corrected subtraction

### PR DESCRIPTION
**Pull request by ashwanthr**

### CODE REVIEW

File: arithmetic.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@ int main()
 
 
     cout << "The sum is: " << c << endl;
-    cout << "The  is: " << c << endl;
+    cout << "The subtraction is: " << c << endl;
```
Comment (ashwanthrs) [inline arithmetic.cpp]: Good catch. Please change it to difference

**Review by ashwanthrs — CHANGES_REQUESTED** [review]
Good catch

**Review by ashwanthr — COMMENTED** [review]
Updated to difference

**Review by ashwanthrs — APPROVED** [review]
Good job

**Pull request merged**